packages/cli/src/config/projects-config.ts

```diff
@@ -13,7 +13,7 @@ import type {
 import { processProjects } from './process-projects';
 import { createDefaultOptions, mergeBuildOptions, validateOptions } from './resolve-options';
 import { resolveNpmRequests } from './resolve-requests';
-import type { ModuleResolver } from '@stylable/core/src/types';
+import type { ModuleResolver } from '@stylable/core/types';
```

Comment:
```suggestion
import type { ModuleResolver } from '@stylable/core/dist/index-internal';
```

you can add the `ModuleResolver` to the existing export in core index-internal